### PR TITLE
Move lime-example to lime-docs, separate lime-docs and lime-docs-minimal

### PR DESCRIPTION
**Pull request by ilario**
This is an alternative to #183 solution to #182 problem.
Here lime-docs and lime-docs-minimal become incompatible, they can't be installed at the same time.
This should break thing as the first is selected by lime-full and the second i selected by lime-basic which is selected by lime-full.
Unexpectedly lime-sdk doesn't complain.
Still this solution is not optimal, so I would suggest either to go for #183 or to avoid using lime-full and lime-basic (related to #178).

### CODE REVIEW

**Comment by ilario** [comment]
Another possibiltiy is that someone fix the 
```
$(INSTALL_DATA) $(filter-out $(PKG_BUILD_DIR)/en_config.txt,$(wildcard $(PKG_BUILD_DIR)/en_*.txt)) $(1)/www/docs/
```
line in lime-docs/Makefile which seems to be broken (refer to #182).

File: packages/lime-docs/Makefile [hunk 1]
```diff
@@ -63,13 +64,15 @@ endef
 
 define Package/$(PKG_NAME)/install
 	$(INSTALL_DIR) $(1)/www/docs/
-	$(INSTALL_DATA) $(filter-out $(PKG_BUILD_DIR)/en_config.txt,$(wildcard $(PKG_BUILD_DIR)/en_*.txt)) $(1)/www/docs/
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/en_*.txt $(1)/www/docs/
+	$(INSTALL_DATA) ./files/lime-example $(1)/www/docs/
```
Comment (gmarcos87) [inline packages/lime-docs/Makefile]: ./files/lime-example not exist on lime-docs folder

Comment (ilario) [inline packages/lime-docs/Makefile]: Wow I failed totally, I moved the lime-example file but I moved it to the wrong place!!!!

Comment (ilario) [inline packages/lime-docs/Makefile]: Fix in #185

**Comment by ilario** [comment]
I moved the lime-example file to the wrong dir, fixed in #185.

**Pull request merged**